src/main/java/uk/gov/hmcts/reform/blobrouter/clients/ErrorNotificationRequest.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.blobrouter.clients;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ErrorNotificationRequest {
+
+    @JsonProperty("zip_file_name")
+    public final String zipFileName;
+
+    @JsonProperty("po_box")
+    public final String poBox;
```

Comment:
could we please add a comment to remove this later once notification service provider makes it non mandatory ?